src/core/op.cc

```diff
@@ -782,7 +782,14 @@ OP(PM)
         // -------------------------------------------------
         ac *l_ac = (ac *)(a_op._reserved_1());
         bool l_match = false;
-        l_match = l_ac->find_first(a_buf, a_len);
+        const bool l_tolower_applied =
+            ((a_ctx->m_src_asn_str.m_tx_applied &
```

Comment:
I wonder if you need the `==` here or if it'll cast to the right bool value depending on if it's zero or non-zero?